SETUP.md

```diff
@@ -88,6 +88,9 @@ For the next step, you need the [Travis command line client](https://github.com/
 This program is a Ruby gem:
 install it with `gem install travis` (not `apt install travis`, which is a different program).
 
+After the install, you will need to provide your credentials to login to travis with
+```travis login --org```
```

Comment:
Can wrap in a single ` instead of the triple ```

Reply:
Let's keep this in the same paragraph as the previous sentence. After switching to single backticks rather than tripple, you can also put that code on the same as the text.

Thanks for helping with doc!

Reply:
@adebali I can merge this if you please make the small changes above.  Thanks again.